<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadForm</class>
 <widget class="QWidget" name="LoadForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="timeTableWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="bottomMargin">
       <number>60</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="timeTableLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;&amp;amp;Timetable:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="buddy">
           <cstring>timeTableLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="timeTableLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnOpenTimeTable">
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="schemeworkWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <property name="bottomMargin">
       <number>60</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="schemeworkLabel">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;&amp;amp;Scheme Of Work:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="buddy">
           <cstring>schemeworkLineEdit</cstring>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="schemeworkLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="btnOpenSchemework">
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnLoadFiles">
     <property name="text">
      <string>Load Files</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>timeTableLineEdit</tabstop>
  <tabstop>btnOpenTimeTable</tabstop>
  <tabstop>schemeworkLineEdit</tabstop>
  <tabstop>btnOpenSchemework</tabstop>
  <tabstop>btnLoadFiles</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
